pkgs/applications/networking/instant-messengers/tangram/default.nix

```diff
@@ -27,15 +45,35 @@ stdenv.mkDerivation rec {
     ninja
     pkg-config
     python3
+    blueprint-compiler
     wrapGAppsHook
   ];
 
+  buildInputs = [
+    gdk-pixbuf
+    gjs
+    glib
+    glib-networking
+    gsettings-desktop-schemas
+    gtk4
+    libadwaita
+    webkitgtk_5_0
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-libav
+    gst-plugins-base
+    (gst-plugins-good.override { gtkSupport = true; })
+    gst-plugins-bad
+  ]);
+
+  dontPatchShebangs = true;
   dontWrapGApps = true;
 
   # Fixes https://github.com/NixOS/nixpkgs/issues/31168
```

Comment:
Is this pointing to the wrong issue?

Reply:
I think this comment is for the postFixup.